C++/Data-Structures/Stacks/Evaluate Postfix Expression.cpp

```diff
@@ -0,0 +1,114 @@
+//This code is contributed by @dalalvarun
+// https://github.com/dalalvarun
+
+#include <iostream>
+#define MAX 40
+using namespace std;
+
+struct stack
+{
+    int data;
+    struct stack *next;
+};
+
+struct stack *top = nullptr;
+struct stack *push(struct stack *, int);
+struct stack *pop(struct stack *);
+int peek(struct stack *);
+
+/*
+ * Trial Output
+ * Enter a postfix expression:2 3 1 * + 9 -
+ * 
+ * The evaluation is: -4
+ */
+
+int main()
+{
+    cout<<"Enter a postfix expression:";
+    char input[MAX];
+    gets(input);
```

Comment:
``gets`` was not declared in this scope. Please fix this by including the required headers.